crates/util/src/paths.rs

```diff
@@ -12,46 +12,55 @@ lazy_static::lazy_static! {
         dirs::config_dir()
             .expect("failed to determine RoamingAppData directory")
             .join("Zed")
+    } else if cfg!(target_os = "linux") {
+        dirs::config_dir()
+            .expect("failed to determine XDG_CONFIG_HOME directory")
+            .join("zed")
     } else {
         HOME.join(".config").join("zed")
     };
-    pub static ref CONVERSATIONS_DIR: PathBuf = CONFIG_DIR.join("conversations");
-    pub static ref EMBEDDINGS_DIR: PathBuf = CONFIG_DIR.join("embeddings");
-    pub static ref THEMES_DIR: PathBuf = CONFIG_DIR.join("themes");
-    pub static ref LOGS_DIR: PathBuf = if cfg!(target_os = "macos") {
-        HOME.join("Library/Logs/Zed")
-    } else if cfg!(target_os = "windows") {
-        dirs::data_local_dir()
-            .expect("failed to determine LocalAppData directory")
-            .join("Zed/logs")
+    pub static ref CONVERSATIONS_DIR: PathBuf = if cfg!(target_os = "macos") {
+        CONFIG_DIR.join("conversations")
     } else {
-        CONFIG_DIR.join("logs")
+        SUPPORT_DIR.join("conversations")
     };
+    pub static ref EMBEDDINGS_DIR: PathBuf = if cfg!(target_os = "macos") {
+        CONFIG_DIR.join("embeddings")
+    } else {
+        SUPPORT_DIR.join("embeddings")
+    };
+    pub static ref THEMES_DIR: PathBuf = CONFIG_DIR.join("themes");
+
     pub static ref SUPPORT_DIR: PathBuf = if cfg!(target_os = "macos") {
         HOME.join("Library/Application Support/Zed")
+    } else if cfg!(target_os = "linux") {
+        dirs::data_local_dir()
+            .expect("failed to determine XDG_DATA_DIR directory")
+            .join("zed")
     } else if cfg!(target_os = "windows") {
-        dirs::config_dir()
-            .expect("failed to determine RoamingAppData directory")
+        dirs::data_local_dir()
+            .expect("failed to determine LocalAppData directory")
             .join("Zed")
     } else {
         CONFIG_DIR.clone()
     };
+    pub static ref LOGS_DIR: PathBuf = if cfg!(target_os = "macos") {
+        HOME.join("Library/Logs/Zed")
+    } else {
+        SUPPORT_DIR.join("logs")
+    };
     pub static ref EXTENSIONS_DIR: PathBuf = SUPPORT_DIR.join("extensions");
     pub static ref LANGUAGES_DIR: PathBuf = SUPPORT_DIR.join("languages");
     pub static ref COPILOT_DIR: PathBuf = SUPPORT_DIR.join("copilot");
     pub static ref DEFAULT_PRETTIER_DIR: PathBuf = SUPPORT_DIR.join("prettier");
     pub static ref DB_DIR: PathBuf = SUPPORT_DIR.join("db");
     pub static ref CRASHES_DIR: PathBuf = if cfg!(target_os = "macos") {
         HOME.join("Library/Logs/DiagnosticReports")
-    } else if cfg!(target_os = "windows") {
```

Comment:
This variable is used for us to detect segfaults and other crashes which are recorded by the system but couldn't be intercepted by us. I don't know if it's possible for us to find this data on linux, but I expect there might be something on Windows.

Reply:
Perhaps we should make it an Option instead, that's only defined for platforms we can find this information on.